Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1237,7 +1237,7 @@ def audit_download_strategy
   def audit_urls
     urls = [url] + mirrors
 
-    if name == "curl" && !urls.find { |u| u.start_with?("http://") }
+    if name == "curl" && !urls.find { |u| u.start_with?("http://") } && url != Formula["curl"].head.url
```

Comment:
Add the `url != Formula["curl"].head.url` into the `urls.find` and maybe split it into a separate variable like `http_non_stable_urls` and also filter out `devel` and do `next if url == Formula["curl"].head.url`.

Reply:
Sure. Happy to take a run at it that way. Thought I'd open with the simplest possible fix & see what the feedback was, since I have something of a habit of overcomplicating solutions right out of the gate 😄.